usautobuild/logger.py

```diff
@@ -51,14 +56,14 @@ def setup_logger(arg_level: str) -> None:
 def setup_extra_loggers(config: Config) -> None:
     """Configure complex loggers requiring config"""
 
-    discord_webhook = config.discord_webhook
-    if discord_webhook is not None:
-        handler = DiscordHandler(discord_webhook)
+    if (discord_webhook := config.discord_webhook) is not None:
```

Comment:
Forgot how to read the walrus, what is the unsugared version of this code?

Reply:
same as before

Reply:
I mean, the walrus is a shorthand for something else. Sugar syntax.

Reply:
Ah I understood what you meant.

Reply:
i just changed non walrus to walrus